pkg/services/storage/plan_schemas.go

```diff
@@ -37,10 +44,20 @@ func generateProvisioningParamsSchema() service.InputParametersSchema {
 			},
 		},
 	}
+	if pd.planName != generalPurposeV1 {
+		ips.PropertySchemas["accessTier"] = &service.StringPropertySchema{
+			Title:         "Access Tier",
+			Description:   "The access tier used for billing.",
+			DefaultValue:  hot,
+			AllowedValues: []string{hot, cool},
+		}
+	}
+	return ips
 }
 
-func generateUpdatingParamsSchema() service.InputParametersSchema {
-	return service.InputParametersSchema{
+// nolint: lll
```

Comment:
Should it be at end the line, or the linter is disabled for the whole function?

Reply:
The linter is disabled for the whole function?